src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig

```diff
@@ -259,7 +259,11 @@
                                                                                 {% endif %}
 
                                                                                 {% if isProduct %}
-                                                                                    {% set attributes = attributes|merge({ itemprop: 'image' }) %}
```

Comment:
Please change the commit from "fix" to "chore" since it doesn't really fix a problem but adds something small to shopware